public/index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="generator" content="Scrivito by Infopark AG (scrivito.com)">
 
   <title>Scrivito Example App JS</title>
+  <script src="custom_js_head.js"></script>
```

Comment:
I'm not 100% satisfied with the naming of the files. How about this?

* `js_snippets_head.js`
* `js_snippets_before_body_end.js`